src/components/pages/ArticlesPage/Cards/Cards.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import s from './Cards.module.css';
+import * as data from '@site/data/resources';
+
+const Card: React.FC<data.Resource> = (props) => {
+  let tagsArray = new Array();
+  if(props.tags) tagsArray = props.tags.split(", ");
```

Comment:
It probably makes sense to change the `Resource['tags']` type from `string` to `string[]`.